.pre-commit-config.yaml

```diff
@@ -42,6 +42,25 @@ repos:
       - id: check-yaml
         name: "Check YAML"
 
+  - repo: https://github.com/psf/black
+    rev: 22.3.0
+    hooks:
+      - id: black
```

Comment:
This would be run every time on CI AFAICS? Perhaps add the manual hook stage option as with spellchecking, to conserve resource?

Reply:
No, it only runs when there's changes to files in the `pep_sphinx_extensions/tests/` directory.